src/drivers/bitbucket_cloud.js

```diff
@@ -0,0 +1,77 @@
+const fetch = require('node-fetch');
+const FormData = require('form-data');
+const { URL, URLSearchParams } = require('url');
+
+const { fetch_upload_data } = require('../utils');
+
+class BitBucketCloud {
+  constructor(opts = {}) {
+    const { repo, token } = opts;
+
+    if (!token) throw new Error('token not found');
+    if (!repo) throw new Error('repo not found');
+
+    this.token = token;
+    this.repo = repo;
+
+    const { protocol, host, pathname } = new URL(this.repo);
+    this.repo_origin = `${protocol}//${host}`;
```

Comment:
double slash

Reply:
is double slash wrong? I assumed it's an escape

Reply:
you're right, it's fine to use single slash. but note that i had copied this code from `gitlab.js`- so same double slash is there

Reply:
Ah sorry its OK!!!! protocol needs double slash!